web/client/components/geostory/contents/Text.jsx

```diff
@@ -29,7 +48,24 @@ export default compose(
     branch(
         ({ mode }) => mode === Modes.EDIT,
         editableText
-    )
-
-
+    ),
+    /*
+     * pass the placeholder localized to the text component
+     * when contentEditing is false (the only case the placeholder is visible)
+    */
+    branch(
+        ({contentEditing}) => !contentEditing,
+        compose(
+            withProps(
+                ({sectionType, contentEditing}) => {
+                    return contentEditing ? {
+                        placeholder: ""
```

Comment:
i think it is safer with "", because of [this ](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/utils/LocaleUtils.js#L144) line

Reply:
see above, I think there is a bug but you should investigate.